examples/src/main/pegasus/org/coursera/example/Course.courier

```diff
@@ -0,0 +1,11 @@
+namespace org.coursera.example
+
+record Course {
+  @related = "instructors.v1"
```

Comment:
Could you remove these? as they should be automatically added by the naptime system.